src/command_modules/azure-cli-policyinsights/azure/cli/command_modules/policyinsights/_params.py

```diff
@@ -0,0 +1,113 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import resource_group_name_type
+
+from azure.cli.command_modules.resource._completers import (
+    get_policy_set_completion_list, get_policy_completion_list,
+    get_policy_assignment_completion_list, get_providers_completion_list, get_resource_types_completion_list)
+
+from ._validators import validate_resource
+
+
+def load_arguments(self, _):
+    for scope in ['state', 'event']:
+        with self.argument_context('policy {}'.format(scope)) as c:
+            c.argument(
+                'management_group_name',
+                options_list=['--management-group-name', '-m'],
+                arg_group='Scope',
+                help='Management group name.')
+            c.argument(
+                'resource_group_name',
+                arg_type=resource_group_name_type,
+                arg_group='Scope')
+            c.argument(
+                'resource',
+                options_list=['--resource', '-r'],
+                validator=validate_resource,
+                arg_group='Resource ID',
+                help='Resource ID or resource name.')
+            c.argument(
+                'namespace',
+                options_list=['--namespace'],
+                completer=get_providers_completion_list,
+                arg_group='Resource ID',
+                help='Provider namespace (Ex: ''Microsoft.Provider'').')
+            c.argument(
+                'resource_type_parent',
+                options_list=['--parent'],
+                arg_group='Resource ID',
+                help='The parent path (Ex: ''resA/myA/resB/myB'').')
+            c.argument(
+                'resource_type',
+                options_list=['--resource-type'],
+                completer=get_resource_types_completion_list,
+                arg_group='Resource ID',
+                help='Resource type (Ex: ''resC'').')
+            c.argument(
+                'policy_set_definition_name',
+                options_list=['--policy-set-definition-name', '-p'],
```

Comment:
`-s` for set would be a much better choice

Reply:
Agreed. Will change it.